deploy/backplane/acs/01-acs-admins-cluster-aggregate.ClusterRole.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-acs-admins-cluster-aggregate
+  labels:
+    managed.openshift.io/aggregate-to-acs-admins: "cluster"
+rules:
+# ACS SRE can manage projects
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - projects
+  verbs:
+  - create
+  - get
+  - list
+  - patch
+  - update
+  - watch
+# ACS SRE can get infrastructure details (cluster)
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - infrastructures
+  - oauths
+  verbs:
+  - 'get'
```

Comment:
```suggestion
  - get
  - list
  - watch
```

Reply:
Why was this not included?  Verbs 'list' and 'watch' are read operations that are generally given with 'get'.  Is there a reason not to include them?